src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
```

Comment:
Might be useful to create a component that stored the buttons and passed up to App.